js/app.js

```diff
@@ -75,40 +111,118 @@ let projects=[
     }
 ];
 
-let div=document.querySelector('.grid-container')
+let project='';
 for(let obj of projects){
     let tech=obj.teachnologies;
-    div+=`<div class="grid-item">
+    project+=`<div class="grid-item project">
     <div class="item-image">
         <img src="${obj.featuredImage}" alt="featured-image">
```

Comment:
- [ ] your project's image are  not showing up  maybe they do in your machines so please try to fix the `src` link by removing a dot like that `"./assets/images/Img-Plaholder.svg"`